### DISCUSSION TITLE
MindMap view in the App.

### DISCUSSION THREAD

**Discussion by sjugraj** [directus/directus#9384] [Feature Requests]
Is there a way to create mindmap visualisation from a collection.
Example:
the table can be:
id | name | parent_id
-- | -- | --
1 | vehicle | null
2 | car | 1
3 | motobike | 1

Here we have the parent information of each node that can help us generate graph in a top-down tree format.
Is there a way to visualise it in the App and do the reverse if possible

like

- if someone create a new node under some existing that becomes the parent of that new row.
- if someone moved the node to under a new parent table reflect the updated information.
- so on..

There are not much mindmap tools out there that support database entries only json files.
example

- https://mindmeister.com
- https://www.mindmapping.com/
- https://drive.mindmup.com/

**Comment by rijkvanzanten** [comment]
No, there's no mind-map layout in Directus (core) currently 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
